REQUIRE

```diff
@@ -7,3 +7,4 @@ Reexport
 FixedSizeArrays
 Measures
 Showoff
+StatsBase
```

Comment:
If we put a future StatPlots version in - could that result in Pkg conflicts when running both on master and installing/updating other packages? I'm not sure if Pkg checks deps for packages on master.